example/android/app/src/androidTest/java/expo/modules/xmtpreactnativesdk/example/EspressoViewFinder.kt

```diff
@@ -16,7 +16,7 @@ import java.util.concurrent.TimeoutException
 
 object EspressoViewFinder {
     private const val CHECK_INTERVAL = 50L
-    private const val TIMEOUT_MS = 10 * 1000L
+    private const val TIMEOUT_MS = 15 * 1000L
```

Comment:
I'm not sure we need this I just bumped it up in fear it was timing out.

Reply:
Reverted!

Reply:
Actually added it back. It seems the extra 5 seconds makes a difference!